database/seeders/DatabaseSeeder.php

```diff
@@ -12,11 +12,15 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // \App\Models\User::factory(10)->create();
-
+        \App\Models\User::factory(10)->create();
```

Comment:
them `use App\Models\User` tren cung.
```suggestion
        User::factory(10)->create();
```